shared-bindings/vectorio/Polygon.c

```diff
@@ -28,12 +28,13 @@
 //|         :param int y: Initial screen y position of the 0,0 origin in the points list."""
 //|
```

Comment:
Missing the new `color_index`

Reply:
Nice catch, thank you. Fixed with latest commit